src/components/Sum.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```

Comment:
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```
Make it easy to read

Reply:
thanks, fixed